src/app/Dashboard/styles/loginPointsModal.scss

```diff
@@ -27,6 +27,19 @@
         transition: all .8s;
         width: 20rem;
     }
+    .dialog {
+        width: 100% !important;
```

Comment:
So much !important use?

Reply:
Yes, there are some inline CSS  in the material UI component, that's the only way for now.